FirebaseSessions/Sources/Settings/LocalOverrideSettings.swift

```diff
@@ -32,7 +32,7 @@ class LocalOverrideSettings: SettingsProvider, SettingsProtocol {
 
   var sessionTimeout: TimeInterval? {
     let timeout = plistValueForConfig(configName: LocalOverrideSettings
-      .PlistKey_sessions_timeout) as? String
+      .PlistKey_sessions_timeout) as? Double
     if timeout != nil {
       return Double(timeout!)
```

Comment:
If we are already casting this as a Double?, we do not have to do typecasting later. Here and below.

Reply:
Moved this to a new PR to show you: https://github.com/firebase/firebase-ios-sdk/pull/10716